tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -446,6 +446,12 @@ test_i2c_hdc: hdc@3f {
 	drdy-gpios = <&test_gpio 0 0>;
 };
 
+test_i2c_hdc2080: hdc2080@40 {
+	compatible = "ti,hdc2080";
+	label = "HDC2080";
+	reg = <0x40>;
+};
```

Comment:
Please add all the variants here.

Reply:
I have done that, and it passes CI.

Reply:
This sounds resolved - @galak - do you have any other concerns?